_layouts/base.html

```diff
@@ -95,7 +95,7 @@
       <header class="usa-header usa-header--extended" role="banner">
           <div class="usa-navbar">
             <div class="usa-logo" id="basic-logo">
-              <a href="{{ site.baseurl }}/" title="Home" aria-label="Home">
```

Comment:
Wouldn't we want to keep `site.baseurl` as much as possible to support links in Federalist previews resolving correctly?

Reply:
yep. we had mixed usage previously and I saw that `identity-site` wasn't using `site.baseurl` (but those appear to just be a few stragglers). Added 6e862e3 to fix it.